src/util/workers.c

```diff
@@ -30,33 +31,58 @@ static void yieldCallback(void *yieldCtx) {
   RedisModule_Yield(ctx, REDISMODULE_YIELD_FLAG_CLIENTS, NULL);
 }
 
+/* Configure here anything that needs to know it can use the thread pool */
+static void workersThreadPool_OnTurnOn(size_t new_num) {
```

Comment:
Is this the right place for these functions?
It doesn't seem natural that the workers are responsible for configurating the module
Consider moving to config.c?

Reply:
The thread pool can also be turned on/off from here due to an event start/end. This logic was also already in this file under `USE_BURST_THREADS` check